.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,49 @@
+---
+name: Feature Request
+about: Suggest an idea for this project
+title: ""
+labels: "feature"
+assignees: ""
+---
+
+## Feature Request
+
+**Describe the feature you'd like:**
+A clear and concise description of the feature you are requesting.
```

Comment:
May I suggest appending the following text with this?
```
Such descriptions can often be expressed in simple stories, like "As a [person], I [want to], [so that]"
```
Please let me know what if you have other suggestions.

Reply:
Sounds good.